src/HydrationRepository.js

```diff
@@ -1,13 +1,17 @@
-class HydrationRepository {
-  constructor(data, id) {
+import UtilityRepository from '../src/UtilityRepository';
```

Comment:
I think Scott had the same question but I'm curious to know why this is being imported? I thought that the `super` on line 5 helped make that connection. I'm glad that it works overall but I do feel a little confused about this subject but I will attempt it in the activity class and see if I can make it work by importing in the file.

Reply:
It seems to not be able to access it's super methods without the import connection.  That solved one of the undefined errors that was coming in.

Reply:
I was getting the same problem but it makes sense now.